daisy_workflows/build-publish/windows_media/windows-install-media.publish.json

```diff
@@ -0,0 +1,38 @@
+{{/*
+  Template to publish Windows install media images.
+  By default this template is setup to publish to the 'gce-image-builder'
+  project, the 'environment' variable can be used to publish to 'test', 'prod',
+  or 'staging'.
+  DeleteAfter is set to 180 days for all environments other than prod where no
+  time period is set.
+*/}}
+{
+  "Name": "windows-install-media",
+  {{$work_project := printf "%q" "gce-image-builder" -}}
+  {{$delete_after := `"24h*30*6"` -}}
+  {{$endpoint := `"https://www.googleapis.com/compute/alpha/projects/"` -}}
+  {{if eq .environment "test" -}}
+  "WorkProject": {{$work_project}},
+  "PublishProject": "bct-prod-images",
+  "ComputeEndpoint": {{$endpoint}},
+  "DeleteAfter": {{$delete_after}},
+  {{- else if eq .environment "prod" -}}
+  "WorkProject": {{$work_project}},
+  "PublishProject": "compute-image-tools",
```

Comment:
We should meet about this. I'd like to understand why this isn't published to windows-cloud. If this can't be in windows-cloud we will likely need a purpose specific project that ends in -cloud.

Reply:
@zmarano I remember we discussed about this before. "windows-cloud" is improper since it's for stable guest images other than a tool image. Is there a existing project ended with "-cloud" that can be a proper host for the "windows-install-media"? If not, I may propose "windows-install-media-cloud".

Reply:
We can change it later since the current version is published to compute-image-tools. Does that work? @bkatyl